src/main/java/jp/kusumotolab/kgenprog/Configuration.java

```diff
@@ -564,12 +582,20 @@ private void setOutDirFromCmdLineParser(final String outDir) {
       this.outDir = Paths.get(outDir);
     }
 
-    @Option(name = "--siblings-count", metaVar = "<num>",
-        usage = "Specifies how many variants are generated from a parent in a generation.",
+    @Option(name = "--mutation-generating-count", metaVar = "<num>",
+        usage = "Specifies how many variants are generated from a generation by a mutation.",
```

Comment:
```suggestion
        usage = "Specifies how many variants are generated in a generation by a mutation.",
```

`--crossover-generating-count` も同様